src/components/CartIcon/CartIcon.jsx

```diff
@@ -0,0 +1,21 @@
+import { useCart } from "../../contexts/CartContext";
+import CartSVG from "../../images/cart.svg";
+import { CartIconContainer } from "./CartIcon.styled";
+
+export default function CartIcon() {
+  const { cart } = useCart();
+
+  let itemsQuantity = 0;
+  cart.forEach(({ quantity }) => {
+    itemsQuantity += quantity;
+  });
+
+  return (
+    <CartIconContainer>
+      <img src={CartSVG} alt="Cart" />
+      {itemsQuantity === 0 || (
+        <span id="badge">{itemsQuantity < 100 ? itemsQuantity : "99+"}</span>
+      )}
```

Comment:
this should just be:
`<span id="badge">{itemsQuantity < 100 ? itemsQuantity : "99+"}</span>`
or am i missing something ? 👀

Reply:
I guess it depends on the expected behavior, I wanted to hid the badge when there were 0 items on the cart:
![image](https://user-images.githubusercontent.com/74197406/146081077-57f9ee5e-2d38-42b6-a5c1-e0eb6cd6ae72.png)
If I remove the conditional rendering then it would look like this:
![image](https://user-images.githubusercontent.com/74197406/146081417-4d1e78ac-24de-4ab2-b685-84dedf40cf5a.png)

Reply:
Ohhhh I see, ok